src/app/(pages)/home/_components/ServerCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import Image from 'next/image';
+import ServerProLogo from '../_assets/server_pro_logo.png';
+
+export default function ServerCard() {
+  let sponsors = [
+    {
+      name: 'Server.pro',
+      description:
+        'Some description of the sponsor here. Some description of the sponsor here. Some description of the sponsor here. Some description of the sponsor here. Some description of the sponsor here.',
+      logo: ServerProLogo,
+    },
+    {
+      name: 'Another Sponsor',
+      description:
+        'Some random descripton. Some random descripton. Some random descripton. Some random descripton. Some random descripton. Some random descripton. Some random descripton. Some random descripton.',
+      logo: ServerProLogo,
+    },
+  ];
+
+  return sponsors.map(sponsor => (
```

Comment:
This implementation works great, however we don't want to do this in the ServerCard component. This is because the ServerCard component should nicely abstract away one individual ServerCard so that the component is reusable. If we add code specific to the SponsorsBlock to this component then that limits it's reusability. Let's move the sponsors object and the sponsors.map call to the SponsorsBlock and pass the sponsor data into this component as a prop. You will have to add a type definition to tell typescript what kind of prop we are expecting but that will make the component more reusable.